md/release-notes.md

```diff
@@ -143,12 +143,66 @@ has been removed.
 ### Fixes in Bonita 7.6
 
 #### Fixes in Documentation
+* BS-17479 IE11 missing from 7.5 list of supported browsers
 
 #### Fixes in Engine component
+* BS-15944	StaleObjectStateException: Row was updated or deleted by another transaction or unsaved-value mapping was incorrect
+* BS-15960	Using REST API to call BDM decreases the performance
+* BS-15975	process instance id and flownode instance id not found when executing the work for the Start event
+* BS-16519	Inconsistent behavior of Bonita Portal for multiple group with same name
+* BS-16595	"My Tasks" view displays executing tasks
+* BS-16791	Can not have two extensions within the same REST API extension definition if one's pathTemplate is a prefix of the other's
+* BS-16868	Robustness: When there is communication issues with database sever, the works are lost
+* BS-16960	Issue with REST Connector stuck on sockRead no default timeout.
+* BS-16969	No license error in the logs when the performance activeProfile is set but the license is not performance
+* BS-16999	Cannot use $ and \ character for password defined in database.properties and use by setup tool
 
 #### Fixes in Studio component
```

Comment:
There is no line break in this section
If you add '*' at the begining of each line, there will be line breaks